index.js

```diff
@@ -1308,14 +1308,14 @@ class GithubScm extends Scm {
      * Get the changed files from a Github event
      * @async  _getChangedFiles
      * @param  {Object}   config
-     * @param  {String}   config.type      Can be 'pr' or 'repo'
-     * @param  {Object}   [config.payload] The webhook payload received from the SCM service.
-     * @param  {String}   config.token     Service token to authenticate with Github
-     * @param  {String}   [config.scmUri]  The scmUri to get PR info of
-     * @param  {Integer}  [config.prNum]   The PR number
-     * @return {Promise}                   Resolves to an array of filenames of the changed files
+     * @param  {String}   config.type               Can be 'pr' or 'repo'
+     * @param  {Object}   [config.webhookConfig]    The webhook payload received from the SCM service.
+     * @param  {String}   config.token              Service token to authenticate with Github
+     * @param  {String}   [config.scmUri]           The scmUri to get PR info of
+     * @param  {Integer}  [config.prNum]            The PR number
+     * @return {Promise}                            Resolves to an array of filenames of the changed files
      */
-    async _getChangedFiles({ type, payload, token, scmUri, prNum }) {
+    async _getChangedFiles({ type, webhookConfig, token, scmUri, prNum }) {
```

Comment:
We also need to fix scm-base.
https://github.com/screwdriver-cd/scm-base/blob/f900e1579391dfd4cdd5ac0f465f56a284265e53/index.js#L173

Reply:
I fixed it at https://github.com/screwdriver-cd/scm-base/pull/90.